Assignment2/juanjoLdlF-JuanJoseLucasdelaFuente.ttl

```diff
@@ -0,0 +1,21 @@
+@base <http://www.ontogrid.net/Ejercicio2#>
+@prefix ejer2: <http://www.ontogrid.net/Ejercicio2#>
+@prefix xsd: <http://www.w3.org/2001/XMLSchema#> 
+
+ejer2:Class01
+	ejer2: includes ejer2:Sensor029
+	ejer2: includes ejer2:Computer101
+	
+ejer2:Computer101
+	ejer2:hasOwner	ejer2:User10A
+
+ejer2:User10A
+	ejer2:hasName	ejer2:"Pedro"^^xsd:string
+	
+ejer2:Sensor029
+	ejer2:hasMeasurement	ejer2:Measurement8401
+
+ejer2:Measurement8401
+	ejer2:hasTemperature "29"^^xsd:int
+	ejer2:atTime "2010-06-12T12:00:12"^^xsd:dataTime
```

Comment:
This is dateTime. It should also finish with ".". I recommend you to check other student submissions that have been already incorporated in the repository to check these small details. Same in the previous line.